onadata/apps/logger/migrations/0006_add-index-to-instance-uuid_and_xform_uuid.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('logger', '0005_instance_xml_hash'),
+    ]
+
+    # This custom migration must be run on Postgres 9.5+.
```

Comment:
@noliveleger, does this really need 9.5? We still have 9.4 in https://github.com/kobotoolbox/kobo-docker/blob/master/base_images/postgres/Dockerfile

Reply:
@jnm My bad. The comment is not accurate anymore. 
I did use a 9.5 option before reverting back to the current migration. 
I forgot to change the comment.